omod/src/main/java/org/openmrs/module/providermanagement/fragment/controller/ProviderEditFragmentController.java

```diff
@@ -312,6 +313,43 @@ else if (ProviderManagementUtils.clearTimeComponent(date).after(new Date())) {
 
     }
 
+    public FragmentActionResult editSupervisor(@RequestParam(value = "supervisee", required = true) Person supervisee,
+                                               @RequestParam(value = "supervisor", required = true) Person supervisor,
+                                               @RequestParam(value = "relationship", required = false) Relationship relationship,
+                                               @RequestParam(value = "date", required = false) Date date,
+                                               @SpringBean("providerManagementService") ProviderManagementService providerManagementService) {
+
+        if (supervisee == null) {
```

Comment:
I think since the supervisee request param is set to true, this can never be null?  Not sure what the use case is for this error message, but if it is possible (but wrong) a submit without supervisee and we want to return this error message, we could make the param required = false